lessons/03_wave/03_03_aBetterModel.ipynb

```diff
@@ -657,7 +659,7 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Quadratic equations, of course, have two solutions. Which should we choose?  It's natural to select the positive root, but in order to determine *which* root is positive, we need to fill in some actual numbers into the model.\n",
+      "Quadratic equations, of course, have two solutions. Here we have to select the positive root, otherwise our model would have an inconsistency. Are you able to see why?  To determine its value, we need to fill in some actual numbers into the model.\n",
```

Comment:
Should we create a short note somewhere explaining why it introduces an inconsistency?  Or we can pin an answer to the forum -- but if we ask an open-ended question like this we should have an answer available.